### PR TITLE
Create filter group nl

### PR DESCRIPTION
**Pull request by haneslinger**
#### Any background context you want to provide?
Create Filter group

### CODE REVIEW

File: seed/migrations/0175_filtergroup.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='FilterGroup',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255)),
+                ('inventory_type', models.IntegerField(choices=[(0, 'Property'), (1, 'Tax Lot')], default=0)),
+                ('query_dict', models.JSONField(default=dict)),
```
Comment (perryr16) [inline seed/migrations/0175_filtergroup.py:21]: query_dict contains filters and sorts?

File: seed/models/filter_group.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+from seed.models import (
+    VIEW_LIST_INVENTORY_TYPE,
+    VIEW_LIST_PROPERTY,
+    Organization,
+    StatusLabel
+)
+
+AND = 0
+LABEL_LOGIC_TYPE = [
+    (AND, 'and'),
```
Comment (perryr16) [inline seed/models/filter_group.py:17]: any reason you couldn't do `(0, 'and')`? is it for the default setting?

File: seed/views/v3/filter_group.py [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+                'message': 'invalid "inventory_type" must be "Property" or "Tax Lot"'
+            }, status=status.HTTP_400_BAD_REQUEST)
+
+        try:
+            label_logic_int = _get_label_logic_int(label_logic)
+        except StopIteration:
+            return JsonResponse({
+                'success': False,
+                'message': 'invalid "label_logic" must be "and", "or", or "exclude"'
+            }, status=status.HTTP_400_BAD_REQUEST)
```
Comment (perryr16) [inline seed/views/v3/filter_group.py:82]: This is just a suggestion, but I believe you could use the following to validate your data. There's a good example in "/views/v3/derived_columns.py"
```
serializer = FilterGroupSerializer(data=body)
serializer.is_valid()
# logic if true
# logic if false
```

File: seed/views/v3/filter_group.py [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+    decorator=swagger_auto_schema_org_query_param)
+@method_decorator(
+    name='retrieve',
+    decorator=swagger_auto_schema_org_query_param)
+@method_decorator(
+    name='update',
+    decorator=swagger_auto_schema_org_query_param)
+@method_decorator(
+    name='destroy',
+    decorator=swagger_auto_schema_org_query_param)
```
Comment (perryr16) [inline seed/views/v3/filter_group.py:39]: does this code auto-generate these endpoints? If so that's clean!

**Comment by haneslinger** [comment]
Closing in favor of a larger PR

**Pull request closed without merging**